packages/fluentui/react-northstar/src/components/Tree/TreeTitle.tsx

```diff
@@ -72,15 +72,21 @@ export interface TreeTitleProps extends UIComponentProps, ChildrenComponentProps
 
   /** Whether or not tree title is selectable. */
   selectable?: boolean;
+
+  /** For selectable parents define if all nested children are checked */
+  incompleteChecked?: boolean;
```

Comment:
Why did you decide not to use `indeterminate`?

Reply:
Changed